src/token.ts

```diff
@@ -120,6 +119,20 @@ export const STAR: Token = {token: 'op', str: '*'};
 export const IMMEDIATE: Token = {token: 'op', str: '#'};
 export const ASSIGN: Token = {token: 'op', str: '='};
 
+// CS -> CS token alias map
+export const CS_TOKEN_ALIAS_MAP = new Map<string, string>([
+  // NOTE: Only synonymous so long as 16-bit is not supported
```

Comment:
We could probably get around this by having a separate map for 8-bit-only aliases, and then having the tokenizer choose which list to use?  Or else expanding the value of this map to a whole object with extra metadata like whether it applies to 8- or 16-bit modes.

I don't recall off the top of my head, but I think the evaluation order is such that the tokenizer could carry the necessary state, and assuming a communication channel between the tokenizer and the assembler, it could probably auto-switch on `rep` and `sep` instructions...?  This is obviously a longer-term change, but it maybe seems worth documenting any preliminary thoughts on the matter, either here (briefly) or somewhere...

Reply:
I figured if 16-bit CPU support was added at that time .addr would be converted into its own token that would be processed differently depending on the current CPU.